### ISSUE TITLE
Error getting source by id: Source [xxxxxx.google_ads] doesn't exist (#400)

### ISSUE THREAD

**Issue by xwaynec** [jitsucom/jitsu#773]
## Environment

- **Deploy method**: `built from sources`
- **Jitsu version**:  for instance `1.38.6`
- **OS**: for instance `Mac OS BigSur`
- **Cloud Provider**: `GCP`

## Description
After connected source of google ads with destination of postgres, I use `sync now` in the google ads. However, the error message is like

<img width="415" alt="截圖 2022-01-27 上午2 20 30" src="https://user-images.githubusercontent.com/59031/151223658-0cd4f1f2-5260-491b-a3eb-ca73067ebeff.png">

## Steps to reproduce
1. create google ads and make sure test connect is fine
2. create postgres and test connect is fine too.
3. enter google ads sync now button then wait the result

## Expected behavior

successfully sync google ads data and store to database.

## Actual behavior

the console log is 

```
2022-01-26 18:15:32 [ERROR]: [response] "/proxy/api/v1/tasks?source=z15156bkmhqhdwtd3viaf6.google_ads&collection=customers&project_id=z15156bkmhqhdwtd3viaf6" error: {"message":"Error getting source: Source [z15156bkmhqhdwtd3viaf6.google_ads] doesn't exist","error":""}
```

**Comment by xtreding** [comment]
Hi @xwaynec! Could you please check application logs (docker logs or log file called `jitsu-main.log` in mounted logs dir)?
There should be an error message `[xxxxxx.google_ads] Error initializing source of type ...`

**Comment by xwaynec** [comment]
I think I found reproduce steps. If I new a source and new template at same time, the collections didn't been written to redis. However, If I create a source first and save it, then edit in it to create collections. It seems to work.

**Comment by absorbb** [comment]
@xwaynec 
I think key of the problem is that UI allows to setup Source without Collections(Stream) while server ignores such Sources. That will be fixed in the next release - it won't be possible to setup Source without collections

**Comment by absorbb** [comment]
Fixed in 1.39.0
